ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js

```diff
@@ -329,6 +359,7 @@ module.exports = function CesiumMap(
           .map(function(result) {
             return _.map(result.getPoints(), function(coordinate) {
               return Cesium.Cartographic.fromDegrees(
+                // fromDegrees takes lon-lat order, not lat-lon; is the array access correct?
```

Comment:
Still needed?

Reply:
I'm jumping back and forth between this function and the new `doPanZoom` function I introduced. When I find the one that works best, the other will be removed. 

On master, this function exists but is empty.

Reply:
Scratch that. This is a note to verify accuracy. I think our arrays are in lat-lon order but the API wants lon-lat.